src/V3SchedReplicate.cpp

```diff
@@ -0,0 +1,259 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Scheduling - replicate combinational logic
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2003-2022 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+//
+//*************************************************************************
+
+#include "config_build.h"
+#include "verilatedos.h"
+
+#include "V3Ast.h"
+#include "V3Error.h"
+#include "V3Global.h"
+#include "V3SenTree.h"
+#include "V3Sched.h"
+#include "V3SplitVar.h"
+#include "V3Stats.h"
+#include "V3Graph.h"
+
+#include <tuple>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+namespace V3Sched {
+
+namespace {
+
+// Driving region flags
+enum Region : uint8_t { INPUT = 0x1, ACTIVE = 0x2, NBA = 0x4 };
+
+//##############################################################################
+// Data structures (graph types)
+
+class Vertex VL_NOT_FINAL : public V3GraphVertex {
+    uint8_t m_drivingRegions = 0;  // The scheduling regions driving this vertex (var or logic)
```

Comment:
```suggestion
    RegionFlags m_drivingRegions{NONE};  // The scheduling regions driving this vertex (var or logic)
```

And add NONE to flags.